tests/migration/migration.go

```diff
@@ -3533,6 +3533,30 @@ func libvirtDomainIsPersistent(virtClient kubecli.KubevirtClient, vmi *v1.Virtua
 	return strings.Contains(stdout, vmi.Namespace+"_"+vmi.Name), nil
 }
 
+func libvirtDomainIsPaused(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) (bool, error) {
```

Comment:
IMHO, we should work with the console and/or look for conditions. Something to work on in the future

Reply:
tbh, if someone is already improving this code, I'd prefer to see an actual improvement rather than just moving the location of a function. 
For example, that explicit `libvirt` name in the name shouldn't be there. Also on L851 we want to know that the domain is running, why are we then checking that it's not paused?
I won't block though.

Reply:
Understood. @vladikr I'll take a look.

Reply:
Okay, do you mean to check if VM is paused by not using `virsh`? I saw there other options, I'm not sure about them - either by calling to [1] or calling to [2].

[1] https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-launcher/virtwrap/manager.go#L193-L196
[2] https://github.com/kubevirt/kubevirt/blob/main/staging/src/kubevirt.io/api/core/v1/types.go#L433-L435

Reply:
@vladikr Hi, can we merge this as it is? Thanks for reviewing it :)